src/prerequisites/tests/test_signals.py

```diff
@@ -154,3 +156,36 @@ def test_update_cache_triggered_by_parent_object_deletion(self, task):
 
         quest.delete()  # doesn't call task because parent_object no longer exists.
         self.assertEqual(task.call_count, 1)
+
+    def test_on_quest_badge_save_with_rank_prereq__creation(self):
+        """ test that creating a prereq with rank requirements makes a new map """
```

Comment:
Better docstrings.  It also checks for alternate (or) prereqs that are Rank.  Use caps when refer to model names, makes cross referencing easier. Prereq, Rank.

Prereqs are complicated, so be precise in your descriptions:

Prereq - the model (a group of conditions)
prereq_object - the actual prerequisite that needs to be met
or_prereq_object - the alternate prerequisite.

so to say "rank requirements" doesn't really mean much.  I think you mean:

> """
> "test that creating a Prereq, where either the `prereq_object` or the `or_prereq_object` are a Rank, results in the generation of a new map (CytoScape) with the Rank as the map's `initial_object`.
> 
> also verifies that nothing breaks if the map already exists.
> """

Reply:
have this as the doc string
```python
"""  Test that creating a Prereq, where either the prereq_object or the or_prereq_object are a Rank, 
results in the generation of a new CytoScape map with the Rank as the map's initial_object.

Prereq - the model (a group of conditions)
prereq_object - the actual prerequisite that needs to be met
or_prereq_object - the alternate prerequisite.
"""
```